bundles/org.openhab.binding.nuki/README.md

```diff
@@ -15,7 +15,7 @@ It is absolutely recommended to configure static IP addresses for both, the open
 ### Nuki Bridge Callback
 
 The Nuki Binding will manage the required callback from the Nuki Bridge to the openHAB server if _manageCallbacks_ is set to `true`.
-If _manageCallbacks_ is not set it will default to `true`.  
+If _manageCallbacks_ is not set it will default to `true`.  Make sure that you've selected the correct network interface in the [network settings](https://www.openhab.org/docs/settings/services_system.html#network-settings).
```

Comment:
Please go to a new line for your new sentence.
And rather "the correct primary address" than "the correct network interface".
It is how the setting is named.

Reply:
Ok, I made a new commit